### PR TITLE
Synergy update to v1.10.1

### PR DESCRIPTION
**Pull request by benf33**
Update to Synergy Core 1.10.1
- review patches
- 64bits build

### CODE REVIEW

**Comment by benf33** [comment]
Ah cr...!! I don't how these commits (7e3ec2a & 
f3c148b) got in, local branch is clean.

**Comment by Mno-hime** [comment]
Maybe you branched off the older tmux branch? One should always branch off from clean & updated `oi/hipster` branch. But surely `git rebase -i HEAD~4` and rebase on top of `upstream/oi/hipster` can fix it.

File: components/desktop/synergy/Makefile [hunk 1]
```diff
@@ -15,14 +15,14 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		synergy
-COMPONENT_VERSION=      1.7.6
+COMPONENT_VERSION=      1.10.1
 COMPONENT_REVISION=     1
```
Comment (Mno-hime) [inline components/desktop/synergy/Makefile:19]: `COMPONENT_REVISION` shouldn't be present on version change.

File: components/desktop/synergy/Makefile [hunk 2]
```diff
@@ -34,10 +34,16 @@ include $(WS_MAKE_RULES)/cmake.mk
 include $(WS_MAKE_RULES)/ips.mk
 
 PATH=$(PATH.gnu)
+PATH=$(PATH.gnu):$(QT5_ROOT)/bin/
+QT5_ROOT=/usr/lib/qt/5.8
```
Comment (Mno-hime) [inline components/desktop/synergy/Makefile:38]: For consistency, switch those two lines, please.

File: components/desktop/synergy/Makefile [hunk 3]
```diff
@@ -46,16 +52,27 @@ MY_INSTALL_32 =            $(BUILD_DIR_32)/.my_installed
 $(MY_INSTALL_32):	$(BUILD_32)
```
Comment (Mno-hime) [inline components/desktop/synergy/Makefile:52]: Can we get rid of 32-bit install, since build is 64-bit?

**Comment by Mno-hime** [comment]
I think sample-manifest.p5m should be regenerated as it still holds `synergyd`.

**Comment by Mno-hime** [comment]
At least Makefile should hold your copyright.

**Pull request closed without merging**